cmake/BLTOptions.cmake

```diff
@@ -61,6 +61,7 @@ option(ENABLE_MPI          "Enables MPI support" OFF)
 option(ENABLE_OPENMP       "Enables OpenMP compiler support" OFF)
 option(ENABLE_CUDA         "Enable CUDA support" OFF)
 
+option(ENABLE_GMOCK        "Enable Google Mock testing support" OFF)
```

Comment:
E.g. it we had 
* ``TESTS_ENABLE_GMOCK`` 
* ``TESTS_ENABLE_GTEST``
* ``TESTS_ENABLE_FRUIT`` 
* ...

Then these would show up in the ``TESTS`` group in ``ccmake``/``cmake-gui`` rather than the ``ENABLE`` group.  
Pro: This would declutter the ``ENABLE`` group
Con: But, it might make the options less discoverable.